report-api/report-templates/eft/eft_instructions_details.html

```diff
@@ -1,4 +1,6 @@
+
 <div class="mb-24 mt-n48 inner-wrap">
+    <p>You need to follow the instructions your financial institution provides you on how to use their EFT service each time you want to send an EFT payment to BC Registries.</p>
```

Comment:
Cool, was talking to Patty about this there are many ways of sending an EFT and varies quite a bit